Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/notification/ShareTestResultNotificationService.kt

```diff
@@ -41,7 +43,7 @@ class ShareTestResultNotificationService @Inject constructor(
         resetPositiveRATTestReminder()
     }
 
-    fun maybeShowSharePositiveTestResultNotification(
+    suspend fun maybeShowSharePositiveTestResultNotification(
```

Comment:
you have appScope here , you can use it and avoid making `maybeShowSharePositiveTestResultNotification` suspended

Reply:
Changed like suggested